internal/armada/server/reporting.go

```diff
@@ -60,22 +59,6 @@ func reportDuplicateDetected(repository repository.EventStore, results []*reposi
 	return nil
 }
 
-func reportDuplicateFoundEvents(repository repository.EventStore, events []*api.JobDuplicateFoundEvent) error {
```

Comment:
we don't need to do this any more because we just generate the duplicate message at submit time and put it into the event sequence